From 02278927df264816130b10a05d9868e1236fc2c7 Mon Sep 17 00:00:00 2001
From: "Ch1llb0/skin.osmc" <36623079+Ch1llb0@users.noreply.github.com>
Date: Mon, 17 Dec 2018 12:11:14 +0100
Subject: [PATCH] Hotfix

Home.xml:
- change onloads to fix first load of new color options

SkinSettings.xml:
- change onloads to fix first load of new color options
---
 16x9/Home.xml         | 6 +++---
 16x9/SkinSettings.xml | 6 +++---
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/16x9/Home.xml b/16x9/Home.xml
index 6f49b03..51d0863 100755
--- a/16x9/Home.xml
+++ b/16x9/Home.xml
@@ -40,9 +40,9 @@
 	<onload condition="String.IsEmpty(Skin.String(color.dialogoverlaynf))">Skin.SetString(color.dialogoverlaynf,None)</onload>
 	<onload condition="String.IsEmpty(Skin.String(color.background))">Skin.SetString(color.background,None)</onload>
 	<onload condition="String.IsEmpty(Skin.String(color.overlay))">Skin.SetString(color.overlay,None)</onload>
-	<onload condition="Skin.HasSetting(BackgroundCustomColour)">Skin.SetString(DefaultColors,no)</onload>
-	<onload condition="Skin.HasSetting(BackgroundCustomColour)">Skin.ToggleSetting(BackgroundCustomColour)</onload>
-	<onload condition="String.IsEmpty(Skin.String(DefaultColorSet))">Skin.SetString(DefaultColorSet,Default)</onload>
+	<onload condition="String.IsEqual(Skin.String(DefaultColorSet),Default)">Skin.SetString(DefaultColorSet,OSMC Blue)</onload>
+	<onload condition="String.IsEmpty(Skin.String(DefaultColorSet))">Skin.SetString(DefaultColorSet,OSMC Blue)</onload>
+	<onload condition="String.IsEmpty(Skin.String(DefaultColors))">Skin.SetString(DefaultColors,no)</onload>
 	<onload condition="String.IsEmpty(Skin.String(DefaultBackgroundOpacity))">Skin.SetString(DefaultBackgroundOpacity,High)</onload>
 	<onload condition="String.IsEmpty(Skin.String(DefaultOverlayOpacity))">Skin.SetString(DefaultOverlayOpacity,High)</onload>
 	<onload condition="String.IsEmpty(Skin.String(OSMCBackgroundOverlay))">Skin.SetString(OSMCBackgroundOverlay,1)</onload>
diff --git a/16x9/SkinSettings.xml b/16x9/SkinSettings.xml
index 2f5d31c..2c3231a 100755
--- a/16x9/SkinSettings.xml
+++ b/16x9/SkinSettings.xml
@@ -26,9 +26,9 @@
 	<onload condition="String.IsEmpty(Skin.String(color.dialogoverlaynf))">Skin.SetString(color.dialogoverlaynf,None)</onload>
 	<onload condition="String.IsEmpty(Skin.String(color.background))">Skin.SetString(color.background,None)</onload>
 	<onload condition="String.IsEmpty(Skin.String(color.overlay))">Skin.SetString(color.overlay,None)</onload>
-	<onload condition="Skin.HasSetting(BackgroundCustomColour)">Skin.SetString(DefaultColors,no)</onload>
-	<onload condition="Skin.HasSetting(BackgroundCustomColour)">Skin.ToggleSetting(BackgroundCustomColour)</onload>
-	<onload condition="String.IsEmpty(Skin.String(DefaultColorSet))">Skin.SetString(DefaultColorSet,Default)</onload>
+	<onload condition="String.IsEqual(Skin.String(DefaultColorSet),Default)">Skin.SetString(DefaultColorSet,OSMC Blue)</onload>
+	<onload condition="String.IsEmpty(Skin.String(DefaultColorSet))">Skin.SetString(DefaultColorSet,OSMC Blue)</onload>
+	<onload condition="String.IsEmpty(Skin.String(DefaultColors))">Skin.SetString(DefaultColors,no)</onload>
 	<onload condition="String.IsEmpty(Skin.String(DefaultBackgroundOpacity))">Skin.SetString(DefaultBackgroundOpacity,High)</onload>
 	<onload condition="String.IsEmpty(Skin.String(DefaultOverlayOpacity))">Skin.SetString(DefaultOverlayOpacity,High)</onload>
 	<onload condition="String.IsEmpty(Skin.String(OSMCBackgroundOverlay))">Skin.SetString(OSMCBackgroundOverlay,1)</onload>
